import-automation/executor/commands/diff_imports.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""cli to diff prod data and dev data generated by import automation tool.
+
+This command does the following:
+1) Trigger a set of imports on both live and dev instances
+2) Wait
+3) Compare the file hashes of what is generated from the trigger.
+
+If there are no diffs, nothing is printed :D
+
+Usage example (take the diffs of 3 imports):
+
+cd import-automation/executor
+
+PYTHONPATH=$(pwd) python commands/diff_imports.py \
+    --abs_import_names=scripts/covid19_india/cases_count_states_data:covid19IndiaCasesCountStatesData,scripts/covid19_india/medical_tests_in_data:covid19IndiaMedicalTestsInData,scripts/ourworldindata/covid19:OWID_COVID19
+"""
+from dataclasses import dataclass
+import time
+from typing import Dict
+
+from absl import flags
+from absl import app
+from google.cloud import storage
+
+from app.executor.cloud_scheduler import trigger_job
+
+_IMPORT_NAME_FORM = '''
+<path to the directory containing the manifest>:<import name>
+'''.strip()
+
+FLAGS = flags.FLAGS
+flags.DEFINE_list(name='abs_import_names',
+                  default=None,
+                  help=('Comma separated list of absolute import names '
+                        'to run the diff on. Follows the format: '
+                        f'{_IMPORT_NAME_FORM}.'),
+                  short_name='i')
+
+flags.DEFINE_string(name='executor_type',
+                    default="GKE",
+                    help=('One of GKE, GAE'))
+
+flags.DEFINE_string(name='base_bucket',
+                    default="datcom-prod-imports",
+                    help=('GCS bucket name of live data'))
+
+flags.DEFINE_string(name='base_bucket_project_id',
+                    default="datcom-204919",
+                    help=('GCP project id for base_bucket'))
+
+flags.DEFINE_string(name='head_bucket',
```

Comment:
nit: rather than base vs. head, prod vs. dev is a bit clearer... but not too particular

Reply:
much clearer, thanks!